scripts/gui_dev.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+while true; do clear; dymoprint_gui; done
```

Comment:
```suggestion
while true; do clear; dymoprint_gui; sleep 1; done
```

Otherwise this may lead to a very fast infinite loop that could prevent Ctrl+C.

Reply:
Fixed